developers/bindings/index.md

```diff
@@ -660,7 +660,7 @@ The following table gives an overview about the main parts of a `DiscoveryResult
 | `bridgeUID` | If the discovered thing belongs to a bridge, the `bridgeUID` contains the UID of that bridge.
 | `properties` | The `properties` of a `DiscoveryResult` contain the configuration for the newly created thing.
 | `label` | The human readable representation of the discovery result. Do not put IP/MAC addresses or similar into the label but use the special `representationProperty` instead. |
-| `representationProperty` | The name of one of the properties which discriminates the discovery result best against other results of the same type. Typically this is a serial number, IP or MAC address. The representationProperty often matches a configuration parameter and is also explicitly given in the thing-type definition. |
+| `representationProperty` | The name of one of the properties or configuration parameters, which discriminates the discovery result against a) other results of the same type, or b) already existing manually created Things. See chapter [Representation Property](#representation-property) below. |
```

Comment:
Try this.  Just a suggestion.
` The name of one of the properties or configuration parameters that best differentiates this result from other results of the same type.`

Reply:
Ok. Done.